justfile

```
@@ -0,0 +1,29 @@
+set quiet
+
+import? '../sdk-codegen/justfile'
```

the trailing `?` on `import` means to swallow the error if this file is missing. So we'll get bonus actions, but external contributors will still be able to use the recipes in this file.

---

I like this.  so a user that does not have codegen will still have access to repo-specific commands but perhaps not any commands that apply to or invoke the code generator?

---

yep, exactly! progressive enhancement, basically